README.md

```diff
@@ -211,7 +211,7 @@ Read more about supported clouds by clicking on the links below:
 ## Support Matrix
 Cloud | Mint | Mint + Remove Admin Cred | Passthrough | Manual | Token
 --- | --- | --- | --- | --- | ---
-AWS | Y | 4.4+ | Y | 4.3+ | 4.6+ (expected)
+AWS | Y | 4.4+ | Y | 4.3+ | 4.6+
```

Comment:
Should be 4.8 when STS token support went GA for AWS.

Reply:
My bad, fixed!